compiler/codegen/mli.cpp

```diff
@@ -220,6 +235,7 @@ void MLIContext::emitClientPrelude(void) {
   }
 
   gen += this->genDefine("CHPL_MLI_IS_CLIENT");
+  gen += this->genHeaderInc(astr(gen_mli_client, ".h"));
```

Comment:
Now that we're adding a header, I wonder if we can't use it to clean up the organization of the other files (not sure yet, just ruminating)?

Reply:
I suspect there are things to move, but would rather save that for a separate PR